Formula/emscripten.rb

```diff
@@ -206,6 +209,8 @@ def caveats
   test do
     # We're targetting WASM, so we don't want to use the macOS SDK here.
     ENV.remove_macosxsdk if OS.mac?
+    # Avoid errors on Linux when other formulae like `sdl12-compat` are installed
+    ENV.delete "CPATH"
```

Comment:
Looks like this worked: https://github.com/Homebrew/homebrew-core/actions/runs/3596817081/jobs/6057988229#step:7:38

This is a no-op on macOS as `ENV.remove_macosxsdk` already deleted `CPATH`. Could put in `if ... else ...` but didn't seem necessary.